arrays/arrays.c

```diff
@@ -36,9 +42,13 @@ Array *create_array (int capacity) {
 void destroy_array(Array *arr) {
 
   // Free all elements
-
+  
```

Comment:
getting rid of the redundant white spaces can reduce the space complexity of the code